package.json

```diff
@@ -149,7 +154,7 @@
     "eslint": "^7.23.0",
     "eslint-import-resolver-typescript": "^3.5.0",
     "eslint-import-resolver-webpack": "^0.13.2",
-    "eslint-plugin-compat": "^4.0.2",
+    "eslint-plugin-compat": "4.0.2",
```

Comment:
intentional?

Reply:
Yeah, had some weird issues and needed to pin. We can unpin in separate work later after more investigation if needed